From ebec713f25b415d8eab7039ddf8afb935dbdd007 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:34:34 -0400
Subject: [alsa] hda/realtek: Add support of ALC891 codec

Message-id: <140972823573-327-patchset-jkysela@redhat.com>
Patchwork-id: 91023
O-Subject: [RHEL 7.1 PATCH 326/442] ALSA: hda/realtek - Add support of ALC891 codec
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

New codec support for ALC891.

Signed-off-by: Kailang Yang <kailang@realtek.com>
Cc: <stable@vger.kernel.org>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: b6c5fbad16aa5026f508093a8d651c25e1cb6179)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/hda/patch_realtek.c b/sound/pci/hda/patch_realtek.c
index 12fb411..246826f 100644
--- a/sound/pci/hda/patch_realtek.c
+++ b/sound/pci/hda/patch_realtek.c
@@ -6227,6 +6227,7 @@ static const struct hda_codec_preset snd_hda_preset_realtek[] = {
  { .id = 0x10ec0670, .name = "ALC670", .patch = patch_alc662 },
  { .id = 0x10ec0671, .name = "ALC671", .patch = patch_alc662 },
  { .id = 0x10ec0680, .name = "ALC680", .patch = patch_alc680 },
+ { .id = 0x10ec0867, .name = "ALC891", .patch = patch_alc882 },
  { .id = 0x10ec0880, .name = "ALC880", .patch = patch_alc880 },
  { .id = 0x10ec0882, .name = "ALC882", .patch = patch_alc882 },
  { .id = 0x10ec0883, .name = "ALC883", .patch = patch_alc882 },
-- 
1.7.1